### DISCUSSION TITLE
Hi, any updates on this project.

### DISCUSSION THREAD

**Discussion by tausifcreates** [Totodore/socketioxide#77] [Q&A] [answered]
The repo has gone quiet for a while. Was wondering if there was any updates.

**Comment by Totodore** [comment, marked as answer]
> The repo has gone quiet for a while. Was wondering if there was any updates.

Hello! 

I'm working full time so it is quite difficult to have the time to improve this lib.

However, starting from Monday, September 18th I will have more time as I work only half a week.

**Reply by tausifcreates** [reply]
Oh, that is very glad to hear. I have very high hopes in this library.
